src/jdk.compiler/share/classes/com/sun/tools/javac/comp/ReflectMethods.java

```diff
@@ -89,18 +57,19 @@
 
 import javax.lang.model.element.Modifier;
 import java.lang.constant.ClassDesc;
+import java.util.List;
 import java.util.*;
 import java.util.function.Function;
 import java.util.function.Supplier;
 
-import static com.sun.tools.javac.code.Flags.NOOUTERTHIS;
-import static com.sun.tools.javac.code.Flags.PARAMETER;
-import static com.sun.tools.javac.code.Flags.SYNTHETIC;
+import static com.sun.tools.javac.code.Flags.*;
 import static com.sun.tools.javac.code.Kinds.Kind.VAR;
-import static com.sun.tools.javac.code.TypeTag.BOT;
-import static com.sun.tools.javac.code.TypeTag.METHOD;
-import static com.sun.tools.javac.code.TypeTag.NONE;
+import static com.sun.tools.javac.code.TypeTag.*;
 import static com.sun.tools.javac.main.Option.G_CUSTOM;
+import static jdk.internal.java.lang.reflect.code.op.CoreOp._new;
+import static jdk.internal.java.lang.reflect.code.op.CoreOp._throw;
+import static jdk.internal.java.lang.reflect.code.type.FunctionType.functionType;
+import static jdk.internal.java.lang.reflect.code.type.JavaType.type;
```

Comment:
Can you revert these? AFAICT they are inconsistently used, plus non-related changes make it harder to review.

Reply:
Agreed - I'll wait to review until these spurious changes are reverted - hard to see the forest from the tree in here :-)